### ISSUE TITLE
Install instructions?

### ISSUE THREAD

**Issue by kronikpillow** [sandsmark/selectdefaultapplication#16]
Well i bumped in to this while I was on OpenSUSE Tumbleweed, now Im on Fedora, and im wondering how to install this since i can't find it on rpmfusion or coprs?

**Comment by crojack** [comment]
You have to compile it from sources. I am trying but I get this error:

[main@comp selectdefaultapplication-master]$ g++ main.cpp widget.cpp -o selectdefaultapplication
In file included from main.cpp:1:
widget.h:4:10: fatal error: QWidget: No such file or directory
    4 | #include <QWidget>
      |          ^~~~~~~~~
compilation terminated.

**Comment by kronikpillow** [comment]
yeah there is no information on dependencies, or how to compile from source ... i tried ... the only place I found this app installable is Arch and OpenSUSE

**Comment by a290** [comment]
`qmake selectdefaultapplication.pro`
then
`make`

In debian package qtbase5-dev must be installed.
